modules/ROOT/pages/upgrade-cluster.adoc

```diff
@@ -0,0 +1,175 @@
+* Applications serving inbound requests should be scaled to a minimum of 2 replicas.
+
+
+[WARNING] Runtime Fabrics running on cluster version 1.0.x will encounter downtime to applications if configured to use an HTTP proxy. An additional step in the procedure is required to apply the HTTP proxy configuration on each node after the upgrade. Please factor this downtime accordingly. The version of Runtime Fabric can be found by running `gravity status` on a node and referencing the cluster version.
+
+## Steps to apply cluster upgrade
+
+. Provision additional worker VMs if required to maintain availability of applications:
+.. If the amount of available resources for application deployments is less than one worker node’s worth, another worker VM with equal CPU, memory, and disk resources should be added to the Runtime Fabric. See xref:manage-nodes.adoc[How to Add a Node to Runtime Fabric].
+
+. Upgrade Runtime Fabric components to the latest version available:
+.. Using a web browser, navigate to Anypoint Runtime Manager and click on the Runtime Fabrics tab.
+.. Find the desired Runtime Fabric and verify there is no upgrade available.
+.. If an upgrade is available, select the Runtime Fabric, select the Manage button, and click “Upgrade to 1.x.x”.
+.. Wait until the upgrade completes. The status should transition to Active after the upgrade has completed.
+.. Keep your web session open as it will be needed in other steps.
+. Verify Runtime Fabric is healthy:
+.. Using your web browser, navigate to the Runtime Fabrics tab in Anypoint Runtime Manager and verify the desired Runtime Fabric is reporting an Active status.
+.. Open a terminal to a Runtime Fabric controller node. Run “gravity status” and confirm the cluster status is healthy.
+.. Keep your terminal open as it’ll be needed in other steps.
+. Download the latest version of the rtfctl command-line utility:
+.. Using your terminal open on a controller node, run the following command: 
++
+----
+sudo curl -L https://anypoint.mulesoft.com/runtimefabric/api/download/rtfctl/latest -o /opt/anypoint/runtimefabric/rtfctl
+----
++
+.. Change file permissions for the rtfctl binary: 
++
+----
+sudo chmod +x /opt/anypoint/runtimefabric/rtfctl
+----
++
+. Get the latest version of the Runtime Fabric installer URL:
+.. Using your web browser, navigate to Anypoint Runtime Manager and click on the Runtime Fabrics tab to view a list of your Runtime Fabrics.
+.. Click the Downloads link on the upper right area of the page.
+.. Select the “Copy Link” icon next to “Installer package” to copy the URL.
+.. Paste the URL in a text editor for reference. 
+. Start the upgrade procedure:
+.. Using your terminal open on a controller node, run the following command, substituting the value of the URL parameter with the “Installer package” URL: 
++
+----
+sudo ./rtfctl upgrade-cluster --url <cluster-installer-url>
```

Comment:
we also support the `--file` parameter, allowing you to pass a file path instead of a url so the package can be pre-downloaded

Reply:
added a note below to include this.